plugins/processors/enum/README.md

```diff
@@ -0,0 +1,27 @@
+# Enum Processor Plugin
+
+The Enum Processor allows the configuration of value mappings for metric fields.
+The main use-case for this is to rewrite status codes such as _red_, _amber_ and
+_green_ by numeric values such as 0, 1, 2. The plugin supports string and bool
+types for the field values. Multiple Fields can be configured with separate
+value mappings for each field.
+
+### Configuration
+Configuration using table syntax:
+`toml
+# Configure a status mapping for field 'status'
+[[processors.enum.fields]]
+  key = "status"
+  [processors.enum.fields.value_mappings]
+    green = 0
+    yellow = 1
+    red = 2
```

Comment:
It might be nice if you could optionally modify the output fieldkey, so that you could transform like this:
```diff
-foo status="red"
+foo status="red",status_code=2i
```

Maybe something like this?:
```toml
[[processors.enum]]
  [[processors.enum.fields]]
    source = "status"
    destination = "status_code"
    [processors.enum.fields.value_mappings]
      green = 0
      yellow = 1
      red = 2
```

Reply:
That is a good idea. I will implement it.